library/psa_its_file.c

```diff
@@ -137,7 +137,8 @@ psa_status_t psa_its_get_info( psa_storage_uid_t uid,
 psa_status_t psa_its_get( psa_storage_uid_t uid,
                           uint32_t data_offset,
                           uint32_t data_length,
-                          void *p_data )
+                          void *p_data,
+                          size_t* p_data_length )
```

Comment:
Please put the `*` next to the variable name, not the type.

Reply:
Done.